### DISCUSSION TITLE
stuck on tutorial

### DISCUSSION THREAD

**Discussion by Sephy19** [voxeltycoon/issues#1011] [General]
Specifically the 2nd step in the tutorial. I am using a laptop but I do not have a mouse. I all what the 2nd step was asking me to do on my bad but it seem I cannot go forward.

**Comment by DanielBarreleiro** [comment]
You can control the camera with the keyboard using W,A,S,D,Q,E
Press Ctrl + W,A,S,D to rotate the camera
After trying out all the combinations the button should turn green
